src/c3chart/CommonND.js

```diff
@@ -133,6 +135,23 @@
             break;
         }
 
+        if(this.y2() instanceof Array && this.y2().length > 0){
```

Comment:
Yeah, I just coded that with "null" in mind... to prevent this.y2().length from breaking